### ISSUE TITLE
pillow have conflicting dependcies

### ISSUE THREAD

**Issue by yexiaohao03** [WASasquatch/was-node-suite-comfyui#468]
The conflict is caused by:
The user requested pillow>=10.3.0
img2texture 1.0.6 depends on pillow<10 and >=9.2

To fix this you could try to:
loosen the range of package versions you've specified
remove package versions to allow pip attempt to solve the dependency conflict

i just don't know how to repair it,
I am using the 秋叶aki integration package,
I know nothing about programming,just a dynamic graphy designer.

**Comment by WASasquatch** [comment]
Yeah img2texture was forked pre-10 and you usually don't say it works for a major version if you don't know. It probably still does work in 10.x. I can take a look.

**Comment by yexiaohao03** [comment]
> Yeah img2texture was forked pre-10 and you usually don't say it works for a major version if you don't know. It probably still does work in 10.x. I can take a look.

thx :)

**Comment by WASasquatch** [comment]
> > Yeah img2texture was forked pre-10 and you usually don't say it works for a major version if you don't know. It probably still does work in 10.x. I can take a look.
> 
> thx :)

Can you try again?

**Comment by yexiaohao03** [comment]
> > > Yeah img2texture was forked pre-10 and you usually don't say it works for a major version if you don't know. It probably still does work in 10.x. I can take a look.
> > 
> > 
> > thx :)
> 
> Can you try again? Be sure to use `--no-cache` so it doesn't just use package it previously downloaded.

awesome work!!!
There are no dependency conflict errors reported anymore.
Thank you so much!
